app/src/main/java/com/github/dodobest/upbitapi/UpbitApplication.kt

```diff
@@ -2,6 +2,18 @@ package com.github.dodobest.upbitapi
 
 import android.app.Application
 import dagger.hilt.android.HiltAndroidApp
+import timber.log.Timber
 
 @HiltAndroidApp
-class UpbitApplication : Application()
\ No newline at end of file
+class UpbitApplication : Application() {
+    override fun onCreate() {
```

Comment:
```suggestion
class UpbitApplication : Application() {

    override fun onCreate() {
```
클래스와 함수의 경계에는 개행으로 구분해주세요.

Reply:
[01746fb](https://github.com/DoTheBestMayB/UpbitAPI/pull/31/commits/01746fb9183498786251cfdcd00af331c63f415b)